pod/perlgov.pod

```diff
@@ -0,0 +1,531 @@
+=encoding utf-8
+
+=head1 NAME
+
+perlgov - Perl Rules of Governance
+
+=head2 PREAMBLE
```

Comment:
This must be removed from the NAME top level section.

Reply:
Done.